packages/website/docs/alpha-1-testnet/start-here.md

```diff
@@ -24,3 +24,4 @@ All the relevant links can be found in this site's navigation. The relevant link
 | [L2 faucet](https://l2faucet.a1.taiko.xyz/)     | For receiving ETH on Taiko A1                      |
 | [L1 explorer](https://l1explorer.a1.taiko.xyz/) | Explore blocks on Ethereum A1                      |
 | [L2 explorer](https://l2explorer.a1.taiko.xyz/) | Explore blocks on Taiko A1                         |
+| [Testnet status](https://status.a1.taiko.xyz/)  | View the current status of the testnet             |
```

Comment:
I'd keep it in the same style as the rest and use:
`[Protocol status]` and `View the protocol status of Taiko A1`

Reply:
Is it accurate to say status of Taiko A1, since it covers the status of the whole Taiko protocol (L1 & L2)

Reply:
A1 stands for alpha 1 and it refers to the taiko protocol, not the L2 specifically 

Maybe a `View the status of the Taiko protocol A1`

Reply:
right, but A1 can get a little confusing since in original links, Taiko A1 descrbies the L2, like with the block explorer and faucet. that's why I went with just testnet, to describe the whole thing. 

doesn't matter to me, whatever is less confusing.